backend/scripts/init_dynamodb.py

```
@@ -0,0 +1,84 @@
+                    ],
+                    "Projection": {"ProjectionType": "ALL"},
+                }
+            ],
+            Tags=[
+                {"Key": "Environment", "Value": "local"},
+                {"Key": "Application", "Value": "meal-planner"},
+            ],
+        )
+
+        print(f"✓ Table created: {table['TableDescription']['TableName']}")
+        print(f"  Primary Key: PK (partition) + SK (sort)")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)